test/pubsub/iam.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*global describe, it, beforeEach */
+
+'use strict';
+
+var assert = require('assert');
+var IAM = require('../../lib/pubsub/iam');
+var noop = function() {};
+
+describe('IAM', function() {
+  var RESOURCE = 'projects/test-project/topics/test-topic';
+  var pubsubMock = {
+    makeReq_: noop
+  };
+  var iam;
+
+  beforeEach(function() {
+    iam = new IAM(pubsubMock, RESOURCE);
+  });
+
+  describe('initialization', function() {
+    it('should localize the resource', function() {
+      assert.strictEqual(iam.resource, RESOURCE);
+    });
+  });
+
+  describe('getPolicy', function() {
+    it('should make the correct API request', function(done) {
+      iam.makeReq_ = function(method, path, q, body) {
+        assert.strictEqual(method, 'GET');
+
+        var expectedPath = RESOURCE + ':getIamPolicy';
+        assert.strictEqual(path, expectedPath);
+
+        assert.strictEqual(q, null);
+        assert.strictEqual(body, null);
+
+        done();
+      };
+
+      iam.getPolicy(assert.ifError);
+    });
+
+    it('should pass the callback the expected params', function(done) {
+      var _policy = {
+        bindings: [{ yo: 'yo' }]
+      };
+
+      iam.makeReq_ = function(method, path, q, body, callback) {
+        callback(null, _policy, _policy);
+      };
+
+      iam.getPolicy(function(err, policy, apiResponse) {
+        assert.ifError(err);
+        assert.deepEqual(policy, _policy);
+        assert.deepEqual(apiResponse, _policy);
```

Comment:
Not a biggy, we still have this all over our tests, but it's best to use `strictEqual` where possible.